alf/examples/ppo2.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Training using PPOAlgorithm.
+
+To run ppo2 on gym CartPole:
+```bash
+python ppo2.py --root_dir=~/tmp/cart_pole_ppo --gin_file=ppo2_cart_pole.gin
+```
+"""
+
+from absl import app
+from absl import flags
+from absl import logging
+
+import gin
+import tensorflow as tf
+
+from alf.examples.off_policy_actor_critic import create_algorithm, create_environment
```

Comment:
No off_policy_actor_critic.py under examples

Reply:
This file should be removed.